hwcomposer/qeglfsscreen.cpp

```diff
@@ -44,11 +44,27 @@
 
 #include <private/qmath_p.h>
 
+#ifdef WITH_SENSORS
+#include <QtSensors/QSensorManager>
+#include <QtSensors/QOrientationSensor>
+#include <QtSensors/QOrientationFilter>
+#include <QtSensors/QOrientationReading>
+#include <qpa/qwindowsysteminterface.h>
+
+#include <QTimer>
+
+#include "moc_qeglfsscreen.cpp"
+#endif
+
 QT_BEGIN_NAMESPACE
 
 QEglFSScreen::QEglFSScreen(HwComposerContext *hwc, EGLDisplay dpy)
     : m_hwc(hwc)
     , m_dpy(dpy)
+    #ifdef WITH_SENSORS
```

Comment:
preprocessor directives should be at column 0

Reply:
Adjusted